rero_ils/permissions.py

```diff
@@ -62,6 +62,23 @@ def staffer_is_authenticated(user=None):
     return None
 
 
+def user_has_roles(user=None, roles=[], condition='or'):
```

Comment:
can you create a test for this a case when you have  two roles as input?

Reply:
done ! Allow to detect a bug in this function and into other permission functions !